Localization: update topic (#5430)
DevExpress/devextreme-documentation: concepts/Common/Localization/01 Dictionaries/05 Add Strings to a Dictionary.md

    }
    export default App;

---

> You can also take a look at this approach in one of our demos:

Reviewer: ```suggestion
You can also see this approach in one of our demos:
```
